VideoQuickStart/ViewController.swift

```diff
@@ -404,6 +482,34 @@ extension ViewController : TVIRemoteParticipantDelegate {
         logMessage(messageText: "Unsubscribed from audio track for Participant \(participant.identity)")
     }
     
+    func subscribed(to dataTrack: TVIRemoteDataTrack,
+                    publication: TVIRemoteDataTrackPublication,
+                    for participant: TVIRemoteParticipant) {
+
+        // We are subscribed to the remote Participant's data Track. We will start receiving the
+        // remote Participant's data messages now.
+
+        logMessage(messageText: "Subscribed to data track for Participant \(participant.identity)")
+        
+        if let remoteDataTrack = publication.remoteTrack {
+            remoteDataTrack.delegate = self
+        }
+    }
+
+    func unsubscribed(from dataTrack: TVIRemoteDataTrack,
+                      publication: TVIRemoteDataTrackPublication,
+                      for participant: TVIRemoteParticipant) {
+
+        // We are unsubscribed from the remote Participant's data Track. We will no longer receive the
+        // remote Participant's data messages.
+
+        logMessage(messageText: "Unsubscribed from data track for Participant \(participant.identity)")
+
+        if let remoteDataTrack = publication.remoteTrack {
+            remoteDataTrack.delegate = nil
+        }
```

Comment:
Should we encourage developers to nil out the delegate? It feels like (very small) added complexity and this is the QS.

Reply:
I can remove.